### PR TITLE
Proofpoint app

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/saas-cloud/proofpoint-on-demand.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+id: proofpoint-on-demand
+title: Proofpoint On Demand
+sidebar_label: Proofpoint On Demand
+description: The Proofpoint On Demand app for Sumo Logic provides an improved security posture by analyzing message logs.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/proofpoint-logo.png')} alt="proofpoint-logo" width="100"/>
+
+Proofpoint On Demand is a cloud-based cybersecurity platform that offers wide range of services to protect businesses against cyber threats. This includes email security, threat intelligence, information protection, and compliance solutions.<br/> The Proofpoint on Demand app for Sumo Logic enables insights into the functioning of email security policies and helps you to gain visibility into the effectiveness of email security policies, therefore making informed decisions to improve security posture. This app will further provide information about the encryption status of the communication.
```
Comment (JainM6) [inline docs/integrations/saas-cloud/proofpoint-on-demand.md:11]:
```suggestion
Proofpoint On Demand is a cloud-based cybersecurity platform that offers a wide range of services to protect businesses against cyber threats. This includes email security, threat intelligence, information protection, and compliance solutions. The Proofpoint on Demand app for Sumo Logic enables insights into the functioning of email security policies and helps you gain visibility into the effectiveness of email security policies, therefore making informed decisions to improve security posture. This app will further provide information about the encryption status of the communication.
```

File: docs/integrations/saas-cloud/proofpoint-on-demand.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+Key features of the Proofpoint On Demand app include: 
+- **Email security monitoring**. Monitor message traffic to detect and prevent spam, phishing, and other email-borne threats.
+- **Compliance monitoring**. Monitor email communications for compliance with internal policies and external regulations such as DLP, DMARC, etc.
```
[the remark this answers is not in the document]
Comment (JainM6) [reply docs/integrations/saas-cloud/proofpoint-on-demand.md:15]: It'd be good if you could add the full form of these terms.

File: docs/integrations/saas-cloud/proofpoint-on-demand.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+- **Incident investigation**. Quickly investigate potential security incidents by searching and analyzing email security and compliance data. This includes identifying the source of a security threat and the extent of its impact.
+- **User behavior monitoring**. Monitor behaviour related to email communication to identify potential insider threats or unauthorized access.
```
Comment (JainM6) [inline docs/integrations/saas-cloud/proofpoint-on-demand.md:17]:
```suggestion
- **User behavior monitoring**. Monitor behavior related to email communication to identify potential insider threats or unauthorized access.
```

Comment (JainM6) [reply docs/integrations/saas-cloud/proofpoint-on-demand.md:17]: US english

**Review by JainM6 — CHANGES_REQUESTED** [review]
Grammar and Formatting changes

File: docs/integrations/saas-cloud/proofpoint-on-demand.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (JainM6) [reply docs/integrations/saas-cloud/proofpoint-on-demand.md]: Please add the full form.

**Review by JainM6 — DISMISSED** [review]
Add the full form please.

File: docs/integrations/saas-cloud/index.md [hunk 4]
```diff
@@ -145,36 +145,42 @@ Learn about the Sumo Logic apps for SaaS and Cloud applications.
   </div>
 </div>
 <div className="box smallbox22 card">
+  <div className="container">
+  <a href="/docs/integrations/saas-cloud/proofpoint-on-demand"><img src={useBaseUrl('img/send-data/proofpoint-logo.png')} alt="icon" width="120"/><h4>Proofpoint On Demand</h4></a>
+  <p>Analyze message logs and provide improved security posture.</p>
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/index.md:150]:
```suggestion
  <p>Analyze message logs and improve your security posture.</p>
```

File: docs/integrations/saas-cloud/proofpoint-on-demand.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+sidebar_label: Proofpoint On Demand
+description: The Proofpoint On Demand app for Sumo Logic provides an improved security posture by analyzing message logs.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/proofpoint-logo.png')} alt="proofpoint-logo" width="100"/>
+
+Proofpoint On Demand is a cloud-based cybersecurity platform that offers a wide range of services to protect businesses against cyber threats. This includes email security, threat intelligence, information protection, and compliance solutions. The Proofpoint on Demand app for Sumo Logic enables insights into the functioning of email security policies and helps you gain visibility into the effectiveness of email security policies, therefore making informed decisions to improve security posture. This app will further provide information about the encryption status of the communication.
+
+Key features of the Proofpoint On Demand app include:
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/proofpoint-on-demand.md:13]:
```suggestion
Key features of the Proofpoint on Demand app include: 
```

File: docs/integrations/saas-cloud/proofpoint-on-demand.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+
+## Installing the Proofpoint On Demand app​
+
+This section has instructions for installing the Proofpoint On Demand app for Sumo Logic and descriptions of each of the dashboards.
+
+Locate and install the app you need from the **App Catalog**. If you want to see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
+1. From the **App Catalog**, search for the app and select it.
+1. Select **Add Integration** button to install the app.
+1. Configure **Proofpoint On Demand** app using the steps described in the [Proofpoint On Demand Cloud-to-Cloud Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/proofpoint-on-demand-source/). If you already have set up your data, skip this step by clicking on **Next**.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/proofpoint-on-demand.md:160]: @JV0812 "on" is lowercase according to their website.

```suggestion
1. Configure **Proofpoint on Demand** app using the steps described in the [Proofpoint on Demand Cloud-to-Cloud Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/proofpoint-on-demand-source/). If you already have set up your data, skip this step by clicking on **Next**.
```

File: docs/integrations/saas-cloud/index.md [hunk 7]
```diff
@@ -145,36 +145,42 @@ Learn about the Sumo Logic apps for SaaS and Cloud applications.
   </div>
 </div>
 <div className="box smallbox22 card">
+  <div className="container">
+  <a href="/docs/integrations/saas-cloud/proofpoint-on-demand"><img src={useBaseUrl('img/send-data/proofpoint-logo.png')} alt="icon" width="120"/><h4>Proofpoint On Demand</h4></a>
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/index.md:149]:
```suggestion
  <a href="/docs/integrations/saas-cloud/proofpoint-on-demand"><img src={useBaseUrl('img/send-data/proofpoint-logo.png')} alt="icon" width="120"/><h4>Proofpoint on Demand</h4></a>
```

File: docs/integrations/saas-cloud/proofpoint-on-demand.md [hunk 8]
```diff
@@ -0,0 +1,187 @@
+---
+id: proofpoint-on-demand
+title: Proofpoint On Demand
+sidebar_label: Proofpoint On Demand
+description: The Proofpoint On Demand app for Sumo Logic provides an improved security posture by analyzing message logs.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/proofpoint-on-demand.md:5]: @JV0812 "on" is lowercase per the product site
```suggestion
title: Proofpoint on Demand
sidebar_label: Proofpoint on Demand
description: The Proofpoint on Demand app for Sumo Logic provides an improved security posture by analyzing message logs.
```

File: docs/integrations/saas-cloud/proofpoint-on-demand.md [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
+
+Key features of the Proofpoint On Demand app include: 
+- **Email security monitoring**. Monitor message traffic to detect and prevent spam, phishing, and other email-borne threats.
+- **Compliance monitoring**. Monitor email communications for compliance with internal policies and external regulations such as Data Loss Prevention (DLP), Domain-based Message Authentication, Reporting and Conformance (DMARC), and other relevant regulations.
+- **Incident investigation**. Quickly investigate potential security incidents by searching and analyzing email security and compliance data. This includes identifying the source of a security threat and the extent of its impact.
+- **User behavior monitoring**. Monitor behavior related to email communication to identify potential insider threats or unauthorized access.
+
+## Log Types
+
+This app uses Proofpoint on Demand source to collect the data from Proofpoint On Demand (PoD) Log Service and uses the secure WebSocket (WSS) protocol, which securely stores the required authentication, scheduling, and state tracking information.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/proofpoint-on-demand.md:21]:
```suggestion
This app uses Proofpoint on Demand source to collect the data from Proofpoint on Demand (PoD) Log Service and uses the secure WebSocket (WSS) protocol, which securely stores the required authentication, scheduling, and state tracking information.
```

File: docs/integrations/saas-cloud/proofpoint-on-demand.md [hunk 10]
```diff
@@ -0,0 +1,187 @@
...
+      * Select **Enter a Custom Data Filter** and type in a custom source category that starts with an underscore. For Example, `_sourceCategory=MyCategory`.
+    2. **Folder Name**. You can retain the existing name, or enter a name of your choice for the app.
+    3. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
+1. Click **Next**.
+
+Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. You can share it with your organization.
+
+The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.
+
+## Viewing Proofpoint On Demand Dashboards​​
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/proofpoint-on-demand.md:173]:
```suggestion
## Viewing Proofpoint on Demand Dashboards​​
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
branding/style

File: docs/integrations/saas-cloud/proofpoint-on-demand.md (hunk 5 above)
Comment (kimsauce) [reply docs/integrations/saas-cloud/proofpoint-on-demand.md:13]: @JV0812 you marked resolved, but it isn't fixed - please click "commit suggestion"

File: docs/integrations/saas-cloud/proofpoint-on-demand.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/integrations/saas-cloud/proofpoint-on-demand.md]: @JV0812 can you please do "Commit suggestion" on Megha's comment above?

[the remark this answers is not in the document]
Comment (JV0812) [reply docs/integrations/saas-cloud/proofpoint-on-demand.md]: Its already updated in the doc.

**Pull request merged**